src/mlpack/core/data/scaler_methods/meannormalization.hpp

```diff
@@ -0,0 +1,102 @@
+/**
+ * @file meannormalization.hpp
+ * @author Jeffin Sam
+ *
+ * MeanNormalization class to scale features.
+ *
+ * mlpack is free software; you may redistribute it and/or modify it under the
+ * terms of the 3-clause BSD license.  You should have received a copy of the
+ * 3-clause BSD license along with mlpack.  If not, see
+ * http://www.opensource.org/licenses/BSD-3-Clause for more information.
+ */
+#ifndef MLPACK_CORE_DATA_MEAN_NORMALIZATION_HPP
+#define MLPACK_CORE_DATA_MEAN_NORMALIZATION_HPP
+
+#include <mlpack/prereqs.hpp>
+
+namespace mlpack {
+namespace data {
+
+/**
+ * A simple Mean Normalization class
+ *
+ * Given an input dataset this class helps you to normalize each
+ * feature.
+ *
+ * @code
+ * arma::Mat input = loadData();
+ * arma::Mat output;
+ *
+ * // Scale the features.
+ * MeanNormalization scale;
+ * scale.Tranform(input, output);
+ *
+ * // Retransform the input.
+ * scale.InverseTransform(output, input);
+ * @endcode
+ */
+class MeanNormalization
+{
+ public:
+  /**
+  * Function to scale Features.
```

Comment:
I think this applies throughout; I think you should use `features` here.

Reply:
changed accordingly.